### ISSUE TITLE
Tooltips still render with quark's improved tooltips

### ISSUE THREAD

**Issue by Lucky-56** [Shadows-of-Fire/Apothic-Attributes#29]
**Forge:**
47.2.30

**Mod version:**
ApothicAttributes-1.20.1-1.3.4

**Other Mods:**
- Placebo-1.20.1-8.6.1
- Quark-4.0-441
- Zeta-1.0-16

![tooltip](https://github.com/Shadows-of-Fire/Apothic-Attributes/assets/55949451/6cc68aac-af04-4bf7-809f-794715d24b87)

nothing much to tell, just to show.

**Comment by Shadows-of-Fire** [comment]
Apoth's modifications don't take well to being displayed with the compressed quark tooltips.  Previously I was just disabling their tooltips on affixed items (and disabling mine on normal items), but I guess since their 1.20 refactor that code isn't functional anymore.

**Comment by Lucky-56** [comment]
yikes so report to quark aswell then?

**Comment by Shadows-of-Fire** [comment]
No, that integration was on my side within Apoth here https://github.com/Shadows-of-Fire/Apotheosis/blob/1.20/src/main/java/dev/shadowsoffire/apotheosis/mixin/client/AttributeTooltipsMixin.java

**Comment by Lucky-56** [comment]
so this would get fixed if
1. I installed Apoth aswell
2. and you got it fixed?

since the issue right now is just attributeslib

**Comment by chromatic-aberration** [comment]
would be cool if this got some more attention - or at the very least, if there was an option to hide attributeslibs tooltips altogether :(

**Comment by Lucky-56** [comment]
> would be cool if this got some more attention - or at the very least, if there was an option to hide attributeslibs tooltips altogether :(

+1
as I wanted to use the mod without apotheosis' adventure module thereby not needing the opened tooltip
EDIT: or without apotheosis alltogether can't remember
